Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -444,6 +489,160 @@ IPCCommandResult SDIOSlot0::SendCommand(const IOCtlVRequest& request)
 
   return GetDefaultReply(return_value);
 }
+
+u32 SDIOSlot0::GetOCRegister() const
+{
+  u32 ocr = 0x00ff8000;
+  if (m_Status & CARD_INITIALIZED)
+    ocr |= 0x80000000;
+  if (m_Status & CARD_SDHC)
+    ocr |= 0x40000000;
+  return ocr;
+}
+
+std::array<u32, 4> SDIOSlot0::GetCSDv1()
```

Comment:
Unfortunately not, because of the `GetSize()` call.
I could take it out, but that would only complicate the function signature.